src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.NonCoreCLR.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Threading
+{
+    //
+    // Portable implementation of ThreadPool
+    //
+
+    public static partial class ThreadPool
+    {
+        private static bool EnsureConfigInitializedCore() => true;
+
+#pragma warning disable IDE0060
+        internal static bool CanSetMinIOCompletionThreads(int ioCompletionThreads) => false;
+        internal static bool CanSetMaxIOCompletionThreads(int ioCompletionThreads) => false;
+#pragma warning restore IDE0060
+
+        [Conditional("unnecessary")]
+        internal static void SetMinIOCompletionThreads(int ioCompletionThreads) { }
+        [Conditional("unnecessary")]
+        internal static void SetMaxIOCompletionThreads(int ioCompletionThreads) { }
```

Comment:
```suggestion
```
Are these actually used for anything?

If these are not used and the only thing left in this file is `EnsureConfigInitializedCore`, it can be `#if CORECLR` ifdef in the main ThreadPool.cs instead.

Reply:
I committed the suggestion but after that, to remove `EnsureConfigInitializedCore` I duplicated the following snippet in both `ThreadPool.Unix.cs` and `ThreadPool.Windows.cs` as I don't see any other file that it's included under the same conditions as the previous one (`FeaturePortableThreadPool=true` and `FeatureCoreCLR=false`). Does that seem fine? Otherwise I believe we would have to keep `ThreadPool.NonCoreCLR.cs`
```C#
#if !CORECLR
        private static bool EnsureConfigInitializedCore() => true;
#endif
```

Reply:
It looks fine to me. There are other one-off lines that are same between .Windows.cs and .Unix.cs files, but it does not seem to be very helpful for readability to try to extract them out into a common file.